packages/ui/primitives/document-dropzone.tsx

```diff
@@ -150,7 +150,7 @@ export const DocumentDropzone = ({
             </motion.div>
           </div>
 
-          <input {...getInputProps()} />
+          <input data-testid="document-dropzone" {...getInputProps()} />
```

Comment:
Although testing by test IDs is the most resilient way of testing, I'm still not very sure about this approach for all tests.

The reason is that testing by test IDs is not user-facing, which means it doesn't resemble how the app is used by the users. Since these are created for testing only, the users don't see and interact with them.

Playwright recommends using this approach when you can't locate by role or text.

[Source](https://playwright.dev/docs/locators#locate-by-test-id)

Looking forward to seeing what others think as well.